src/cryptoadvance/specter/__main__.py

```diff
@@ -17,7 +17,6 @@
 from os import path
 import signal
 
-DEBUG = True
```

Comment:
We will reestablish the default with this:
If you do a
FLASK_ENV=development
debug will be true by default.
If you want degub to be true in production, you can do:
FLASK_DEBUG=True